From 87ba17e2c9ffbe058a4a77be5f7e1c49c293a040 Mon Sep 17 00:00:00 2001
From: Robert Mader <robert.mader@posteo.de>
Date: Tue, 20 Dec 2022 20:36:51 +0100
Subject: [PATCH 375/463] arm64: dts: rk3399-pinephone-pro: Add camera
 orientations

Marking the cameras as front and back cameras. This allows downstream to
better name the devices (e.g. "Build-in Front Camera"), selecting
appropriate cameras depending on use-cases and apply e.g. mirroring for
front cameras.

Signed-off-by: Robert Mader <robert.mader@posteo.de>
---
 arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
index 43e38c58675ce..f2437625226d2 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
@@ -772,6 +772,7 @@ wcam: camera@1a {
 		reset-gpios = <&gpio1 RK_PA0 GPIO_ACTIVE_LOW>;
 		powerdown-gpios = <&gpio2 RK_PD4 GPIO_ACTIVE_HIGH>;
 
+		orientation = <1>;
 		rotation = <270>;
 
 		lens-focus = <&wcam_lens>;
@@ -812,6 +813,7 @@ ucam: camera@36 {
 		rockchip,camera-module-name = "CameraKing";
 		rockchip,camera-module-lens-name = "Largan-9569A2";
 
+		orientation = <0>;
 		rotation = <90>;
 
 		port {
-- 
2.46.0

